src/web/rustdoc.rs

```diff
@@ -361,7 +364,57 @@ fn path_for_version(req_path: &[&str], target_name: &str, conn: &Connection) ->
     } else {
         req_path[3]
     };
-    format!("{}?search={}", crate_root, search_item)
+    format!("{}/?search={}", crate_root, search_item)
```

Comment:
This breaks https://github.com/rust-lang/docs.rs/issues/502.

Reply:
reverted